dockers/docker-database/database_config.json.j2

```diff
@@ -53,10 +53,25 @@
             "separator": "|",
             "instance" : "redis"
         },
-        "RESTAPI_DB" : {
+        "GB_ASIC_DB" : {
             "id" : 8,
             "separator": "|",
             "instance" : "redis"
+        },
+        "GB_COUNTERS_DB" : {
+            "id" : 9,
+            "separator": "|",
+            "instance" : "redis"
+        },
+        "GB_FLEX_COUNTER_DB" : {
+            "id" : 10,
+            "separator": "|",
+            "instance" : "redis"
+        },
+        "RESTAPI_DB" : {
+            "id" : 11,
```

Comment:
let's keep RESTAPI_DB as number 8.

Reply:
https://github.com/Azure/sonic-swss-common/blob/646053f0a0f95b4b8a7c092654786a0582ce4ad1/common/schema.h

Reply:
https://github.com/Azure/sonic-swss-common/pull/385

Reply:
I abandoned the above, someone already beat me to it, please merge .

Reply:
I've abandoned this, Azure/sonic-swss-common#386 needs to be approved/merged and has same changes, from another contributor.